cmd/zstreamdump/zstreamdump.c

```diff
@@ -665,8 +665,8 @@ main(int argc, char *argv[])
 			}
 			if (verbose) {
 				(void) printf("WRITE_EMBEDDED object = %llu "
-				    "offset = %llu length = %llu\n"
-				    "    toguid = %llx comp = %u etype = %u "
+				    "offset = %llu length = %llu"
```

Comment:
Looks like you're missing a space before the `"` here.

Reply:
fixed